Sources/NIOHTTP1Server/main.swift

```diff
@@ -33,6 +33,18 @@ extension String {
     }
 }
 
+private func httpResponseHead(request: HTTPRequestHead, status: HTTPResponseStatus, headers: HTTPHeaders = HTTPHeaders()) -> HTTPResponseHead {
```

Comment:
Subtle bug here: this doesn't actually emit the headers passed into `headers:`: it quietly ignores them.

Reply:
😳 thanks

Reply:
fixed